hc: 0.263
gravity: 9.80665
dt: 0.01
control_frequency: 100
joint_state_topic: "/nao_raisim_ros/joint_states"
com_topic: "/nao_raisim_ros/CoM"
odom_topic: "/nao_raisim_ros/odom"
zmp_topic: "/nao_raisim_ros/ZMP"
wrenchLLeg_topic: "/nao_raisim_ros/LLeg/force_torque_states"
wrenchRLeg_topic: "/nao_raisim_ros/RLeg/force_torque_states"
LLegodom_topic: "/nao_raisim_ros/LLeg/odom"
RLegodom_topic: "/nao_raisim_ros/RLeg/odom"
gait_phase_topic: "/nao_raisim_ros/gait_phase"
joint_cmd_topic: "/mocapnet/joint_state"
#CoM Admittance Control Parameters
enabled: True
CoM_admittance_gain_x: 0.05
CoM_admittance_gain_y: 0.05
ZMP_proportional_gain_x: 1.5
ZMP_proportional_gain_y: 1.5
DCM_proportional_gain_x: 5.0
DCM_proportional_gain_y: 5.0
DCM_Integral_gain_x: 0
DCM_Integral_gain_y: 0

foot_damping_control_gain: 0.0000
foot_damping_control_time_constant: 0.01
base_damping_control_gain: 0.0000
base_damping_control_time_constant: 0.01


Linear_task_gain: 0.15 # 15 
Angular_task_gain: 0.1 # 25.0 0.005
CoM_task_weight: 1.0
LLeg_task_weight: 15.0
RLeg_task_weight: 15.0
Torso_task_weight: 1.0e-2
Head_task_weight: -1
LHand_task_weight: -1
RHand_task_weight: -1
DOF_weight: 5.0e-5
DOF_gain: 0.45 #0.25
